www/src/components/ecosystem/ecosystem-section.js

```diff
@@ -1,100 +1,94 @@
 import EcosystemFeaturedItem from "./ecosystem-featured-item"
 import { mediaQueries } from "gatsby-design-tokens/dist/theme-gatsbyjs-org"
 
-const EcosystemSectionRoot = styled(`section`)`
-  background: ${p => p.theme.colors.card.background};
-  padding: 0 ${p => p.theme.space[6]};
-  margin-bottom: ${p => p.theme.space[3]};
-
-  ${mediaQueries.md} {
-    box-shadow: ${p => p.theme.shadows.raised};
-    border-radius: ${p => p.theme.radii[2]};
-    display: flex;
-    flex-basis: calc(50% - ${p => p.theme.space[5]});
-    flex-direction: column;
-    flex-grow: 0;
-    margin: 0 ${p => p.theme.space[2]} ${p => p.theme.space[6]};
-    max-height: 60vh;
-    padding: ${p => p.theme.space[6]};
-    padding-bottom: 0;
-
-    :last-child {
-      flex-grow: 1;
-    }
-  }
-
-  ${mediaQueries.lg} {
-    flex-basis: calc(33.33% - ${p => p.theme.space[5]});
-    max-height: 100%;
-
-    :last-child {
-      align-self: flex-start;
-      padding-bottom: ${p => p.theme.space[6]};
-    }
-  }
-
-  a {
-    text-decoration: none;
-  }
-`
-
-export const Header = styled(`header`)`
-  align-items: flex-start;
-`
-
-const Title = styled(`h1`)`
-  align-items: center;
-  color: ${p => p.theme.colors.heading};
-  display: flex;
-  font-size: ${p => p.theme.fontSizes[4]};
-  font-weight: ${p => p.theme.fontWeights.heading};
-  line-height: ${p => p.theme.lineHeights.solid};
-  margin: 0;
-  margin-bottom: ${p => p.theme.space[1]};
-  min-height: ${p => p.theme.space[7]};
+const ecosystemSectionRootStyles = {
+  bg: `card.background`,
+  px: 6,
+  mb: 3,
+  [mediaQueries.md]: {
+    boxShadow: `raised`,
+    borderRadius: 2,
+    display: `flex`,
+    flexBasis: t => `calc(50% - ${t.space[5]})`,
+    flexDirection: `column`,
+    flexGrow: 0,
+    mx: 2,
+    mb: 6,
+    maxHeight: `60vh`,
+    p: 6,
+    pb: 0,
+    ":last-child": {
+      flexGrow: 1,
+    },
+  },
+  [mediaQueries.lg]: {
+    flexBasis: t => `calc(33.33% - ${t.space[5]})`,
+    maxHeight: `100%`,
+    ":last-child": {
+      alignSelf: `flex-start`,
+      pb: 6,
+    },
+  },
+  a: {
+    textDecoration: `none`,
+  },
+}
 
-  span {
-    margin: 0 ${p => p.theme.space[1]} 0 0;
-  }
-`
+export const Header = props => (
+  <header {...props} sx={{ alignItems: `flex-start` }} />
+)
 
-const Icon = styled(`span`)`
-  display: block;
-  height: ${p => p.theme.space[7]};
-  width: ${p => p.theme.space[7]};
-`
+const titleStyles = {
+  alignItems: `center`,
+  color: `heading`,
+  display: `flex`,
+  fontSize: 4,
+  fontWeight: `heading`,
+  lineHeight: `solid`,
+  m: 0,
+  mb: 1,
+  minHeight: t => t.space[7],
+  span: {
+    mr: 1,
+  },
+}
 
-const SubTitle = styled(`h2`)`
-  color: ${p => p.theme.colors.lilac};
-  font-size: ${p => p.theme.fontSizes[1]};
-  font-weight: normal;
-  letter-spacing: ${p => p.theme.letterSpacings.tracked};
-  margin: 0;
-  margin-top: ${p => p.theme.space[5]};
-  text-transform: uppercase;
-`
+const iconStyles = {
+  display: `block`,
+  height: t => t.space[7],
```

Comment:
theme-ui has a `sizes` property that you can define if you want to add to `gatsby-design-tokens.` Until then maybe it's better to keep these as explicit px/rem values?

Reply:
Yeah, would have been best. Establishing a relation to `space` was too tempting 😅